components/console-backend-service/Dockerfile

```diff
@@ -1,43 +1,52 @@
-FROM golang:1.11.4-alpine3.8 as builder
+FROM golang:1.13.11-alpine3.11 as builder
 
 ENV BASE_APP_DIR /go/src/github.com/kyma-project/kyma/components/console-backend-service
+
+ENV GO111MODULE on
+ENV CGO_ENABLED 0
+
+
 WORKDIR ${BASE_APP_DIR}
 
-#
-# Copy files
-#
+COPY ./go.mod .
+COPY ./go.sum .
+
+COPY ./internal ./internal
+COPY ./pkg ./pkg
+COPY ./main.go .
+COPY ./Makefile .
```

Comment:
Would it be possible to just copy everything?

Reply:
It would be but would also make creating a `.dockerignore` file necessary. We suggest (and commited) another soultion - 4 `COPY` lines instad of 6 😃